### ISSUE TITLE
No keyboard input while the `Get-Credential` dialog is open

### ISSUE THREAD

**Issue by jaapvandervelde** [microsoft/terminal#11185]
I read issue #4448 and checked that the `Touch Keyboard and Handwriting Panel Service` is running correctly (restarts, etc.)

However I still have this issue: if I'm running a Terminal (PowerShell) and I start a second one with this command:
```
Start-Process -FilePath "powershell" -Credential $(Get-Credential -Message 'Domain admin password required.' -UserName <some valid username>) -ArgumentList '-NoExit', '-Command', "cd $(Get-Location)"
```
A new PowerShell window opens, but the behaviour of it is very odd. Keyboard input doesn't seem to arrive at it, but when I type something (for example `whoami` to check it worked), it does show up *after* I switch focus away from the new window. The same behaviour applies to the Terminal Windows I started it from. It now only shows entered input after switching focus away from it.

If I close either window, the other will start working normally again immediately. I do not have this issue when starting a separate PowerShell window from the Start menu, regardless of whether I start it as myself, as local admin or as the same other user.

Note that the user account I'm starting the new PowerShell as is a domain admin account, while the account running the original Terminal window is not. Not sure if it matters, I don't have a spare non-domain admin account on hand to try if the same problem occurs when using a non-admin account.

**Comment by zadjii-msft** [comment]
Hmmmmm. There's something weird about the way the credential manager window is interacting with the console window here. It's like, blocking the Console window from being focused, like it's a `MessageBox` or something. I wonder if it does something _extra_ weird when the console is a conpty without a real window.

**Comment by zadjii-msft** [comment]
Okay I think this is fixed as of 33b8a20e1. Or at least, I can't seem to repro it anymore. Admittedly, I can't exactly repro this on 1.12 either. I don't think I ever see the "new" powershell window open. The credential dialog does open, but there's no other console window around failing to receive input.
